src/AddTodoForm.js

```diff
@@ -5,7 +5,7 @@ const AddTodoForm = () => {
     <div>
         <form>
             <label htmlFor='todoTitle'> TITLE </label> <br /> <br />
-            <input type='text' id='todoTitle' value={"Time to add new tasks!"} />  <br /> <br />
+            <input type='text' id='todoTitle'/>  <br /> <br />
             <button type='submit'> Add  </button>
             <button type='submit'> Delete </button> <br /> <br />
```

Comment:
You should change the "submit" type of the Delete button to "button" (to avoid conflicts with the add submit button). Alternatively, you may want to consider removing the Delete button altogether, as it seems you might want a dedicated delete button for each individual todo item in the future.

Reply:
I think I got the point. Because when we're adding it'll add up to the list. However, if we hit the delete button, it doesn't know which entry should be deleted, so it needs a dedicated delete button for each. Thanks